repo/packages/wannier90/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Wannier90(MakefilePackage):
+    """Wannier90 calculates maximally-localised Wannier functions (MLWFs)"""
+
+    url = "https://github.com/wannier-developers/wannier90/archive/refs/heads/develop.tar.gz"
+
+    maintainers("AndresOrtegaGuerrero")
+
+    license("GPL-3.0-or-later", checked_by="AndresOrtegaGuerrero")
+
```

Comment:
Versions are required, otherwise Spack doesn't know which source archive to fetch. Example from [the original recipe](https://github.com/spack/spack-packages/blob/develop/repos/spack_repo/builtin/packages/wannier90/package.py):

```suggestion
    version("develop", branch="develop")
    version("3.1.0", sha256="40651a9832eb93dec20a8360dd535262c261c34e13c41b6755fa6915c936b254")
    version("3.0.0", sha256="f196e441dcd7b67159a1d09d2d7de2893b011a9f03aab6b30c4703ecbf20fe5b")
    version("2.1.0", sha256="ee90108d4bc4aa6a1cf16d72abebcb3087cf6c1007d22dda269eb7e7076bddca")
    version("2.0.1", sha256="05ea7cd421a219ce19d379ad6ae3d9b1a84be4ffb367506ffdfab1e729309e94")
```